osf/admin.py

```diff
@@ -6,48 +6,33 @@
 
 class NodeAdmin(ForeignKeyAutocompleteAdmin):
     fields = (
-        'title',
-        'description',
+        'affiliated_institutions',
```

Comment:
True, I only modified this because `fields` was already there, and a whitelist is safer than a blacklist.